src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -811,39 +811,27 @@
   </ItemGroup>
   <ItemGroup>
     <IoSources Include="$(BclSourcesRoot)\System\IO\__Error.cs" />
-    <IoSources Include="$(BclSourcesRoot)\System\IO\__HResults.cs" />
```

Comment:
I do not think __HResults should be deleted. It is necessary to make things compile.